frontend/src/components/Banner.tsx

```diff
@@ -3,16 +3,16 @@ import { OutboundLink } from "react-ga";
 import { useLocalization } from "@fluent/react";
 import styles from "./Banner.module.scss";
 import { useLocalDismissal } from "../hooks/localDismissal";
-import { CloseIcon, InfoFilledIcon } from "./Icons";
+import { CloseIcon, WarningFilledIcon, InfoFilledIcon } from "./Icons";
 import { useGaViewPing } from "../hooks/gaViewPing";
 
 export type BannerProps = {
   children: ReactNode;
-  type?: "promo" | "warning";
+  type?: "promo" | "warning" | "info";
   title?: string;
   illustration?: ReactNode;
   cta?: {
-    target: string;
+    target?: string;
```

Comment:
This is causing the CI build failure because line 77 passes `target` to `OutboundLink.to` when it might be `undefined`. Are we intending to pass `target` to `OutboundLink.to`? There's also an `OutboundLink.target` field - maybe we should be keeping these things aligned so `cta.target` -> `OutboundLink.target`?

If not, and we want to keep this code the way it is, we can just remove this `?` optional property to make it required. All the calls to `Banner` pass a value to it. (Though one of them passes an empty string `""`)
```suggestion
    target: string;
```

Reply:
I couldn't remove it because a sub component requires that prop. I added a new `btn` type in place of `cta` for this specific use case within Banner.